tests/extract_tests.ml

```diff
@@ -8,8 +8,7 @@ let protect ~finally f =
       finally ();
       r
 
-let disabled =
-  [ 175; 184; 185; 410; 411; 414; 415; 416; 428; 468; 469; 516; 536 ]
+let disabled = [ 175; 184; 185; 516; 536 ]
```

Comment:
This is where the only merge conflict is arising! Let me know if you have any trouble resolving the conflict here.